companies/phoenixcontact.json

```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "phoenixcontact",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Phoenix Contact",
```

Comment:
The imo relevant legal names in germany are "PHOENIX CONTACT GmbH & Co. KG" and "PHOENIXCONTACT Deutschland GmbH", both have the same address.